app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,9 @@
+class CommentsController < ApplicationController
+  def create
+    post = Post.find(params[:id])
+    comment = current_user.comments.new(text: params[:text])
+    comment.post = post
+    comment.save!
+    redirect_to user_post_path
+  end
```

Comment:
- [ ] Here try to use the `if else` condition so that if it doesn't save, your app doesn't crash and you can give a custom message and redirect to a certain path.

```ruby
if comment.save
 ...
else
 ...
end
```